ecs/cloudformation.go

```diff
@@ -43,6 +43,10 @@ func (b *ecsAPIService) Convert(ctx context.Context, project *types.Project) ([]
 		return nil, err
 	}
 
+	if c, ok := project.Extensions[extensionCluster]; ok {
```

Comment:
Any reason to add this here rather than in b.convert() ? 
And maybe that will integrate better with unit tests, add a unit test on this new metedata field

Reply:
nope, will change this